### ISSUE TITLE
Failed to use debugger.

### ISSUE THREAD

**Issue by LittleFall** [Jatana/FastOlympicCoding#39]
![image](https://user-images.githubusercontent.com/30543181/112681811-2c139f00-8eaa-11eb-985b-8ebc393d4ec8.png)

![image](https://user-images.githubusercontent.com/30543181/112681938-52393f00-8eaa-11eb-9b6a-eca0ba5f042a.png)

**Comment by LittleFall** [comment]
macOS 11.2.1
xcode-select version 2384
lldb-1200.0.44.2

**Comment by Jatana** [comment]
Could you run `python --version` from the terminal and paste the result here

**Comment by LittleFall** [comment]
> Could you run `python --version` from the terminal and paste the result here

```
Python 2.7.16
```

**Comment by LittleFall** [comment]
gently push for some updates...

**Comment by Mariosz2** [comment]
@LittleFall Have you solved this problem?👀

**Comment by LittleFall** [comment]
> @LittleFall Have you solved this problem?👀

not yet...

**Comment by Mariosz2** [comment]
all right, thank you.

**Comment by zhk1211** [comment]
@Jatana So how to use it?

**Comment by zhk1211** [comment]
@LittleFall Have you solved it till now?

**Comment by LittleFall** [comment]
no, I use CLion now, still thanks the author of this repo, it help many of my friends.

**Comment by LittleFall** [comment]
> @LittleFall Have you solved it till now?

no, I use CLion now, still thanks the author of this repo, it help many of my friends.
